features.md

```diff
@@ -0,0 +1,86 @@
+HTMLAttributesRenderer
+----------------------
+
+This feature allows you to write Markdown that will render 508 compliant html attributes.
```

Comment:
It would be nice with an explanation (maybe a link) of what "508 compliant" means. It's just a googling away but still.

Reply:
See (48c8134477d1cc829a54e221d767635c8dbdb673)